spec/v1/bundle.md

```diff
@@ -2,40 +2,59 @@
+An attestation Bundle is a collection of multiple attestations in a single
 file. This allows attestations from multiple different points in the software
 supply chain (e.g. Provenance, Code Review, Test Result, vuln scan, ...) to
 be grouped together, allowing users to make decisions based on all available
 information.
 
-**NOTE**: The bundle is not authenticated as a whole. Instead each individual
-attestation is authenticated using [DSSE]. As such, an attacker might be able
-to _delete_ an attestation without being detected. Predicates that follow
-[the monotonic principle] should not have any issues with this behavior.
+**NOTE**: The Bundle is not authenticated as a whole. Instead each individual
+attestation is authenticated using signature formats like [DSSE]. As such,
+an attacker might be able to _delete_ an attestation without being detected.
+Predicates that follow [the monotonic principle] should not have any issues
+with this behavior.
 
 ## Data structure
 
 Attestation Bundles use [JSON Lines] to store multiple attestations.
 
--   Each attestation within a bundle MAY have a different signing key,
+-   Each attestation within a Bundle MAY have different signing keys,
     `_type`, `subject`, and/or `predicateType`.
--   Each line SHOULD be an [Envelope]. Consumers MUST ignore unrecognized
-    lines.
+-   Each line within a Bundle SHOULD be an [Envelope]. Consumers MUST ignore
+    unrecognized lines.
 -   Consumers MUST ignore attestations with unrecognized keys, types,
     subjects, or predicates.
--   Processing of a bundle MUST NOT depend on the order of the attestations.
+-   Processing of a Bundle MUST NOT depend on the order of the attestations.
 
 ## File naming convention
 
 Bundles SHOULD use the suffix `.intoto.jsonl`.
 
-A bundle of attestations relevant for `<filename>` SHOULD be named
-`<filename>.intoto.jsonl`. Attestations in the bundle MAY have different
-subjects, but they SHOULD all be relevant to that file or its dependencies.
-For example, a package named `foo-1.2.3.tar.gz` with hash `abcd` that was
-built from git commit `1234` could have a bundle name
-`foo-1.2.3.tar.gz.intoto.jsonl` with two attestations, one with subject
-`abcd` and one with subject `1234`.
+-   A Bundle of attestations relevant for `<filename>` SHOULD be named
+    `<filename>.intoto.jsonl`.
+-   Attestations in the Bundle MAY have different subjects, but they SHOULD
+    all be relevant to that file or its dependencies.
+
+### Example
+
+A package named `foo-1.2.3.tar.gz` with hash `abcd` that was built from git
+commit `1234` could have a Bundle name `foo-1.2.3.tar.gz.intoto.jsonl` with
+two attestations, one with subject `abcd` and one with subject `1234`.
+
+## Storage convention
+
+The media type `application/vnd.in-toto.bundle` SHOULD be used to denote
+a Bundle in arbitrary storage systems.
+
+-   The encoding of the Bundle contents MAY be omitted from its media type
```

Comment:
Are there any systems that _require_ an encoding in the media type? If not, maybe the `MAY` is too permissive here. Why allow for two media types at all?

Reply:
I don't know of any systems that require the encoding, but this is a good question. We weren't really thinking of this as allowing to media types given the implicit expectation for jsonl, but I do realize that some systems match string literals as opposed to parsing the media type string. So, I'm fine with erring on the side of being explicit here. 

@TomHennen Any input here?

Reply:
Sorry for the late reply.  I don't really have an opinion and don't know enough about media types to say.

Is the suggestion that we just not allow the encoding of the bundle contents at all?

(I bet we can fix this in a follow on PR if we want)

Reply:
Yes. My understanding is that we need the encoding `+jsonl` when there are different encodings to choose from. If the bundle is always `+jsonl`, do we need it? I'll say that I'm not an expert with media types either.

Reply:
Also, this isn't a blocker for the PR I think.

Reply:
If the consensus is that Bundles currently don't support encodings that aren't jsonl, I'm fine with dropping this line in the spec altogether. It's quick enough fix.